ztp/internal/data/cluster/objects/090-baremetalhost.yaml

```diff
@@ -10,6 +10,9 @@ metadata:
   annotations:
     inspect.metal3.io: disabled
     bmac.agent-install.openshift.io/hostname: {{ .Hostname }}
+{{ if eq .Kind "Worker" }}
+    bmac.agent-install.openshift.io/role: worker
+{{ end }}
```

Comment:
Would be nice if you can indent these lines:

```
    {{ if eq .Kind "Worker" }}
    bmac.agent-install.openshift.io/role: worker
    {{ end }}
```

It makes the template easier to read, in my opinion.

Reply:
Sorry, bad habit from helm templates. Let me do that